_uw-research-computing/user-expectations.md

```diff
@@ -35,10 +35,27 @@ to delete data from any CHTC data location at at any time, to preserve
 systems performance, and are not responsible for data loss or file system
 corruption, which are possible in the absence of back-ups.
 
-**Note: CHTC is not HIPAA-compliant** and users should not bring HIPAA data into 
+**CHTC is not HIPAA-compliant** and users should not bring HIPAA data into 
 CHTC services. If you have data security concerns or any questions about 
 data security in CHTC, please get in touch! 
```

Comment:
```suggestion
data security in CHTC, [please get in touch](https://chtc.cs.wisc.edu/uw-research-computing/get-help.html)! 
```

Not part of your changes but a direct link to how to contact us would probably he helpful